### PR TITLE
<Stepper/> - focus & enter step

### PR DESCRIPTION
**Pull request by mosheKerbel**
Stepper should be accessible through tab & enter as you would expect it to be.

### CODE REVIEW

**Comment by mosheKerbel** [comment]
@jonathanadler

File: src/Stepper/Step/Step.js [hunk 1]
```diff
@@ -36,16 +36,23 @@ class Step extends React.PureComponent {
...
+    const { type, active, id, text, onFocus } = this.props;
     const { stepHover } = this.state;
-
     return (
       <div
         data-type={DataHooks.step}
         data-step-type={type ? type : STEP_TYPES.NORMAL}
         data-active={active ? ACTIVE_STEP : ''}
         key={`step${id}`}
+        onKeyUp={e => this._onKeyUp(e, id)}
```
Comment (jonathanadler) [inline src/Stepper/Step/Step.js]: You can use:
```
onKeyUp={this._onKeyUp}
```

File: src/Stepper/Step/StepCircle/StepCircle.st.css [hunk 2]
```diff
@@ -92,25 +87,25 @@
 
 .text {
   -st-extends: Text;
-  -st-states: disabled(boolean), selected(boolean), error(boolean), stepHover(boolean), completed(boolean);
+  -st-states: disabled, selected, error, stepHover, completed;
```
Comment (jonathanadler) [inline src/Stepper/Step/StepCircle/StepCircle.st.css]: This line can be removed now

**Comment by mosheKerbel** [comment]
Merging
failed on an applitools test of the stepper that we really changed.

**Pull request merged**